release-content/0.14/migration-guides/13159_Improve_tracing_layer_customization.md

```diff
@@ -1,28 +1,40 @@
-The `LogPlugin`’s `update_subscriber` field has been replaced with the more flexible `custom_layer`.
+Bevy uses `tracing` to handle logging and spans through `LogPlugin`. This could be customized with the `update_subscriber` field, but it was highly restrictive. This has since been amended, replacing the `update_subscriber` field with the more flexible `custom_layer`. which returns a `Layer`.
```

Comment:
```suggestion
Bevy uses `tracing` to handle logging and spans through `LogPlugin`. This could be customized with the `update_subscriber` field, but it was highly restrictive. This has since been amended, replacing the `update_subscriber` field with the more flexible `custom_layer`, which returns a `Layer`.
```

Reply:
I will open a PR to fix this up in a bit.

Reply:
I got it in #1426, no worries!